text/0069-api-v2.md

```diff
@@ -236,7 +236,30 @@ More about `GC` of deleted/expired entries:
 
 ### Backup and Restore
 
-*To be supplemented in another PR*
+Due to the significant change on storage format, only if the existing TiKV cluster is empty or storing only `TiDB` data, users can enable or disable API v2 smoothly. In other scenario, we need implement one tool to migrate data, which is called [TiKV-BR].  
+
+[TiKV-BR] forks from [TiDB BR] and need some improvements:
+- Storage data conversion from `API V1`(with or without [TTL]) to `API V2`.
+- Backup supports return `backup-ts` in `API V2` TiKV cluster. `TiKV-CDC` can use `backup-ts` as `start-ts` when [creating change feed].
+
+#### Api Version Conversion
+
+To support api version conversion, a parameter `dst-api-version` is introduced in [TiKV-BR] and pass it to TiKV stores in [`BackupRequest`].
+During the backup process, TiKV stores do scanning on all raw key-value entries and do conversion between current `api-version` and `dst-api-version`, then write the key-value entries with the `dst-api-version` format to [SST] files. Restoration does not need rewriting and conversion work, which can speedup the restoration process.
+
+#### Backup Ts
+
+`backup-ts` only takes effect in the `API V2` cluster, which is defined as a `TSO` where all data written before it is backed up to [SST] files.  
+The key challenge is that every TiKV store pre-fetches and caches batch `TSO` from `PD`, so the current `TSO` from PD does not satisfy the `backup-ts` requirements.  
+
+The process to get `backup-ts` is as following:
+1. Get current `TSO` in [TiKV-BR] at the beginning of backup.
+2. Flush cached `TSO` in every TiKV store before scanning process during backup.
+3. Minus `safe-interval` from current `TSO`.
+
+The third process is introduced because the **inflight** RawKV entries are using the ts before the current `TSO`. Even if the cached `TSO`s are flushed, the current `TSO` still can't satisfy the `backup-ts` requirements. The `safe-interval` is a safe enough time duration in which the **inflight** writing RawKV IO can finish. This is an empirical value and default be `1m`, which is safe enough even in a high-pressure system.
```

Comment:
```suggestion
The third step is introduced because there would be **inflight** RawKV entries with timestamp before the current `TSO`. The `safe-interval` is a safe enough time duration that during which **inflight** writes should have finished. This is an empirical value and defaults to 1 minute, which is safe enough even in a high-pressure system.

Besides, this process can be optimized after we implement `safe-ts` for RawKV in stale read scenario, which is a timestamp that all RawKV writes before this timestamp can be seen **safely**.
```